src/frontend/src/components/login/login.component.tsx

```diff
@@ -3,44 +3,44 @@ import * as React from 'react'
 export class LoginComponent extends React.Component {
     public render() {
         return(
-            <form
-                className="login-form"
-                id="LoginForm"
-                aria-label="Record Expunge Login Form">
-                <label className="title h2">
-                    Record Expunge
-                </label>
-                <div className="form-group">
-                    <label htmlFor="name"
-                           className="f6 b db mb2">
-                        Email
-                    </label>
-                    <input id="name"
-                           className="w-100 mb4 pa3 br2 b--black-10"
-                           type="text"
-                           aria-describedby="name-desc"/>
-                </div>
-                <div className="form-group">
-                    <label htmlFor="password"
-                           className="f6 b db mb2">
-                        Password
-                    </label>
-                    <input className="w-100 mb4 pa3 br2 b--black-10"
-                           type="password"
-                           id="password"
-                           aria-describedby="password-desc"/>
-                </div>
-                <div className="form-group buttons">
-                    <button
-                        type="submit"
-                        className="f4 link dim br2 ph3 pv2 mb2 dib white bg-dark-blue">
-                        Log In
-                    </button>
-                    <a className="f4 fw6 db black link hover-hot-pink" href="#">
-                        Forgot Password?
-                    </a>
-                </div>
-            </form>
+            <section className="cf mt4 mb3 pa4 pa5-ns pt4-ns bg-white shadow br3">
+                <form
+                    className="login-form"
+                    id="LoginForm"
+                    aria-label="Record Expunge Login Form">
+                    <IconComponent iconName="logo"/>
```

Comment:
` <IconComponent iconName="logo"/>`  This Component needs an import statement up top to make it available to this file. 

e.g. 
```typescript
import { IconComponent } from '../common/icon.component';
```

Reply:
thanks! I'm not sure how it was working before 🤔